docs/tutorial/code/flask/task.yaml

```diff
@@ -0,0 +1,212 @@
+###########################################
+# IMPORTANT
+# Comments matter!
+# The docs use the wrapping comments as
+# markers for including said instructions
+# as snippets in the docs.
+###########################################
+summary: Getting started with Flask tutorial
+
+kill-timeout: 90m
+
+environment:
+
+execute: |
+  #move everything to $HOME so that Juju deployment works
+  mv *.yaml *.py *.txt $HOME
+  cd $HOME
+  #don't use the staging store for this test
+  unset CHARMCRAFT_STORE_API_URL
+  unset CHARMCRAFT_UPLOAD_URL
+  unset CHARMCRAFT_REGISTRY_URL
+  #make sure that Juju version is correct
+  snap install rockcraft --classic
+  snap install lxd
+  lxd init --auto
+  snap refresh charmcraft --channel=latest/edge --amend
+  snap install microk8s --channel=1.31-strict/stable
+  #adduser $USER snap_microk8s
+  #newgrp snap_microk8s
+  snap refresh juju --channel=3.5/stable --amend
+  lxc network set lxdbr0 ipv6.address none
+  mkdir -p ~/.local/share
+  microk8s status --wait-ready
+  microk8s enable hostpath-storage
+  microk8s enable registry
+  microk8s enable ingress
+  juju bootstrap microk8s dev-controller
```

Comment:
Some general coding tips:

* Leave a space between `#` and the first character of a comment for readability
* Add blank lines between groups of code statements a bit like paragraphs to make them easier to read. Its a judgement call on which statements to group, I would try and find groups of 3-10 lines that are logically similar